spec/controllers/subclassed_pages_controller_spec.rb

```diff
@@ -1,34 +1,34 @@
-require 'spec_helper'
+require "spec_helper"
 
 describe SubclassedPagesController do
   render_views
 
-  describe 'on GET to /subclassed_pages/also_exists' do
-    before { get :show, :id => 'also_exists' }
+  describe "on GET to /subclassed_pages/also_exists" do
+    before { get :show, id: "also_exists" }
```

Comment:
Ah, neat!

---
Teo Ljungberg

> On 8 juli 2016, at 16:56, Damian Galarza <notifications@github.com> wrote:
> 
> In spec/controllers/subclassed_pages_controller_spec.rb:
> 
> >  
> >  describe SubclassedPagesController do
> >    render_views
> >  
> > -  describe 'on GET to /subclassed_pages/also_exists' do
> > -    before { get :show, :id => 'also_exists' }
> > +  describe "on GET to /subclassed_pages/also_exists" do
> > +    before { get :show, id: "also_exists" }
> In a previous PR for testing against the beta I added a shim for handling this -
> 
> https://github.com/thoughtbot/high_voltage/blob/master/spec/support/http_method_shim.rb
> 
> Which was borrowed from Clearance.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>